packages/tools/cli/src/index.ts

```diff
@@ -1,10 +1,5 @@
-import {
-  Commands,
-} from './models';
-
-import {
-  Executor,
-} from './utils';
+export { Commands } from './models';
```

Comment:
It seems to be working here/

Reply:
The previous comment should cover this, let me know if this is a different question, thanks!